### PR TITLE
Audit of Available Logs

### PR DESCRIPTION
**Pull request by ronaldheft-usds**
As part of our security review, we have to send our logs to Splunk (#358). This is the first pass at making a list of all available logs in our system, which we will need to send over to Splunk.

## Changes
- Lists out all known logs in Azure.

## Checklist
- [ ] Tested locally?
- [ ] Added new dependencies?
- [x] Updated the docs?
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## To Be Done

- Once we have a list of all the logs, comb through them and verify no PII/PHI.

### CODE REVIEW

**Comment by ronaldheft-usds** [comment]
@cglodosky @jimduff-usds I made a first pass at documenting out all of our system logs. Could the both of you verify I'm not missing any logs? Is there anyone else I should pull into this review?

File: prime-router/docs/azure-logs-audit.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+| Front Door         | Network requests                        | TBD (Do we have PII/PHI as request params?) | Disabled                     | No        | ?                          | We should enable these logs                             |
+| WAF                | Firewall denials                        | No                                          | No WAF enabled, so no logs   | No        | ?                          |                                                         |
+| primedatahub*      | Access errors, etc                      | No                                          | Resource > Logs              | No        | ?                          |                                                         |
+
+
+## Potential PII/PHI in Application Logs
+
+1. Batch function logs entire message when received.
+2. Send function logs retry and failed messages.
+3. Transports can log failures of files. Any concerns of PII in them?
```
Comment (jimduff-usds) [inline prime-router/docs/azure-logs-audit.md:18]: OK, so I went off on the following wild goose chase:   Knowing that sftp fails in Test, I went to recreate a failure there, so I could answer this question with actual text from a log.  This led me to see that sftp was never happening in Test, which led me to discover a bug in our newest release.  So - good work!   You helped find a major bug!  ;)

Comment (MauriceReeves-usds) [reply prime-router/docs/azure-logs-audit.md:18]: Digging through the source for `net.schmizz.sshj` it looks like the `IOException` thrown by the `SftpClient` logs the file name, but nothing else.

The other concern would be the connect method for the `java.net.Socket` method. That might merit some additional review.

File: prime-router/docs/azure-logs-audit.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Potential PII/PHI in Request Logs
+
+1. Download function has a request parameter `file`. Could the file URL have PII in it?
+2. Action history checks on some parameters `key` and `code`. Neither look like PII, but would like to verify.
```
[the remark this answers is not in the document]
Comment (ronaldheft-usds) [reply prime-router/docs/azure-logs-audit.md]: Created #451 to track this issue.

File: prime-router/docs/azure-logs-audit.md (hunk 1 above)
Comment (ronaldheft-usds) [reply prime-router/docs/azure-logs-audit.md]: Haha, glad I could help find that @jimduff-usds.

**Pull request merged**